api/envoy/config/cluster/v3/cluster.proto

```diff
@@ -737,7 +737,11 @@ message Cluster {
   // is respected by both the HTTP/1.1 and HTTP/2 connection pool
   // implementations. If not specified, there is no limit. Setting this
   // parameter to 1 will effectively disable keep alive.
-  google.protobuf.UInt32Value max_requests_per_connection = 9;
+  //
+  // .. attention::
+  //   This field has been deprecated in favor of the :ref:`max_requests_connection <envoy_api_field_config.core.v3.HttpProtocolOptions.max_requests_connection>` field
+  google.protobuf.UInt32Value max_requests_per_connection = 9
+      [deprecated = true, (envoy.annotations.disallowed_by_default) = true];
```

Comment:
I don't think we should deprecate and disallowed_by_default in the same PR
`disallowed_by_default` should be added in next release cycle

Reply:
Reverted.